lib/helper/Polly.js

```diff
@@ -65,15 +68,20 @@ class Polly extends Helper {
   }
 
   // Start mocking network requests/responses
-  async _startMocking(title = 'Test') {
-    if (!(this.helpers && this.helpers.Puppeteer)) {
-      throw new Error('Puppeteer is the only supported helper right now');
+  async startMocking(title = 'Test') {
+    if (!(this.helpers && (this.helpers.Puppeteer || this.helpers.WebDriver))) {
+      throw new Error('Puppeteer and WebDriver are the only supported helpers right now');
+    }
+    if (this.helpers.Puppeteer) {
+      await this._connectPuppeteer(title);
+    } else if (this.helpers.WebDriver) {
+      await this._connectWebDriver(title);
     }
-    await this._connectPuppeteer(title);
   }
 
   // Connect Puppeteer helper to mock future requests.
   async _connectPuppeteer(title) {
```

Comment:
Requires a simple docblock, as this method can be executed from other helpers. And it will be available in documentation...